[RFC] add some sanity for interface-changes.rst (#13587)
mpv-player/mpv: DOCS/release-policy.md

> - Update `DOCS/client-api-changes.rst` and `DOCS/interface-changes.rst`
>   (in particular, update the last version numbers if necessary)
> 
> - When updating `DOCS/interface-changes.rst`, use the
>   `TOOLS/gen-interface-changes.py` script to generate the new file and only
>   manually edit it if necessary.
> 
> - After updating `DOCS/interface-changes.rst`, delete all stray `.txt` files in
>   the `DOCS/interface-changes` directory except for `example.txt`.
> 

Reviewer: less is more
```suggestion
- Update `DOCS/client-api-changes.rst` (in particular, update the last version
  number if necessary)

- Run `TOOLS/gen-interface-changes.py` to refresh `DOCS/interface-changes.rst`,
  edit manually as necessary.

- Delete all `.txt` files in the `DOCS/interface-changes` directory except for `example.txt`.

```
